spec/observables/fromEvent-spec.js

```diff
@@ -114,17 +114,17 @@ describe('Observable.fromEvent', function () {
       }
     };
 
-    function selector(x) {
-      return x + '!';
+    function selector(x, y, z) {
```

Comment:
Would you able to expand test cases instead of modifying it, such as testing
- emitted none via fromEvent (zero maybe?)
- emitted 1 via fromEvent
- emitted >1 via fromEvent